spec.md

```diff
@@ -398,6 +398,34 @@ Example error response (invalid `id` provided):
 
 ### Resource Data Model
 
+A resource is a named (addressable) blob stored in a Space, with metadata.
+The data model is derived from [W3C FileAPI: File 
+Interface](https://w3c.github.io/FileAPI/#file-section), but with the addition
+of a few crucial properties.
+
+In similar storage systems, a resource is called "File", "Object", "Document",
+"Row", "Graph", and so on.
+
+Resource properties:
+
+* `id`
+* `contents` (Blob)
+* `name` - optional
+* (tbd) a link to a **system-meta** auxiliary object (determined by the local
```

Comment:
Can we say "Links to any metadata objects controlled by the DataPub server"

allowing for many
and not necessarily trying to introduce/define any notion of a specific system-meta auxiliary object vs a link relation type to 0 or more objects.

Reply:
Sure, I like that.